storybook/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build-storybook": "build-storybook",
     "start": "start-storybook -p 9009",
-    "clean": "rimraf node-modules package-lock.json"
+    "clean": "rimraf node-modules"
```

Comment:
What's the difference between `rimraf` and `rm -rf`, do we know?

Reply:
I had always been using it out of habit, so thanks for finally making me look it up!  So it's cross-platform (eg: works on windows) which might not be something we need, and it's faster.  I had always thought that it was safer (than rm -rf on an environment I didn't know anything about) but it seems like that's not the case.  
I'm happy keeping it or not

Reply:
From cursory googling it seemed like `rm -rf` and I didn't understand the difference, but those all sound like great reasons to keep it.

Going to resolve this.